README.md

```diff
@@ -74,6 +74,44 @@ Then update the `package.json` file in your project with the new version:
   }
 ```
 
+### Working with the Pattern Library
+
+Employer Style Base includes a [pattern library](http://cb-talent-development.github.io/employer-style-base/) built with [Fractal](http://fractal.build/). The pattern library serves as a quick reference to the components that come with style base and how to use them.
+
+If you haven't already, from within the employer-style-base repo run
+
+```
+yarn install
+```
+
+This will install Fractal and Fractal CLI.
+
+In order to show styled components, you must first compile from `.scss` to `.css`. Run
+
+```
+npm dev
```

Comment:
it should have said `npm run dev` instead of `npm dev`. Now I'll change it to `yarn run dev`. I haven't used either of the individual `build` and `watch` scripts, just `npm run dev` which does both